lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runtime/azbatch/util/AzureBatchFileNames.java

```diff
@@ -27,7 +27,7 @@
  * Access to the various places things go according to the REEF Azure Batch runtime.
  */
 @Private
-public final class AzureBatchFileNames {
+public final class AzureBatchFileNames extends REEFFileNames {
```

Comment:
Why is this change necessary?

Reply:
So I could call this.azureBatchFileNames.getLocalFolderPath() in ShimManager without Injecting REEFFileNames.